frontend/src/lib/lemon-ui/LemonSelectMultiple/LemonSelectMultiple.tsx

```diff
@@ -18,20 +18,20 @@ export interface LemonSelectMultipleOptionItem extends LemonSelectMultipleOption
 
 export type LemonSelectMultipleOptions = Record<string, LemonSelectMultipleOption>
 
-export interface LemonSelectMultipleProps {
+export type LemonSelectMultipleProps = {
     selectClassName?: string
     options?: LemonSelectMultipleOptions | LemonSelectMultipleOptionItem[]
-    value?: string[] | null | LabelInValue[]
+    value?: string | string[] | null
     disabled?: boolean
     loading?: boolean
     placeholder?: string
     labelInValue?: boolean
-    onChange?: ((newValue: string[]) => void) | ((newValue: LabelInValue[]) => void)
     onSearch?: (value: string) => void
     onFocus?: () => void
     onBlur?: () => void
     filterOption?: boolean
     mode?: 'single' | 'multiple' | 'multiple-custom'
+    onChange?: ((newValue: string) => void) | ((newValue: string[]) => void)
```

Comment:
below we say `const typedOnChange = onChange as (newValue: string | null) => void` should this be `((newValue: string) => void) | ((newValue: string[]) => void) | ((newValue: null) => void)`?

(I have lazily only read the code in the diff 🙈)